db/migrate/20170609100310_migrate_public_body_data.rb

```diff
@@ -0,0 +1,19 @@
+# -*- encoding : utf-8 -*-
+class MigratePublicBodyData < ActiveRecord::Migration
```

Comment:
As far as I can tell, the data is _already_ mirrored to the translation tables. I'm not saying we don't want to do this to be sure, but I don't know if you found cases where it was _necessary_ to do this.